bokeh/models/plots.py

```diff
@@ -56,12 +57,8 @@ def _select_helper(args, kwargs):
         selector = kwargs
     return selector
 
-class LayoutBox(Model):
-    ''' Represents an **on-canvas** layout.
-
-    '''
 
-class Plot(Component):
+class Plot(LayoutDOM):
```

Comment:
Perhaps related is that I'd like LayoutDOM (or whatever it's to be called) to also be private and not exposed to users and to be put into /core/layout in the same place as LayoutCanvas but it seemed to be necessary to make it public....I think because of the check in io.py - if you have any suggestions, in this area, that would be great.

(note I previously put this note in the wrong place)